src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -90,9 +90,10 @@ subroutine CStateUpdateDynPatch(bounds, num_soilc_with_inactive, filter_soilc_wi
     ! TODO(wjs, 2017-01-02) Do we need to move some of the FATES fluxes into here (from
     ! CStateUpdate1) if use_fates is true? Specifically, some portion or all of the fluxes
     ! from these updates in CStateUpdate1:
-    ! cf_soil%decomp_cpools_sourcesink_col(c,j,i_met_lit) = cf_soil%FATES_c_to_litr_lab_c_col(c,j) * dt
-    ! cf_soil%decomp_cpools_sourcesink_col(c,j,i_cel_lit) = cf_soil%FATES_c_to_litr_cel_c_col(c,j) * dt
-    ! cf_soil%decomp_cpools_sourcesink_col(c,j,i_lig_lit) = cf_soil%FATES_c_to_litr_lig_c_col(c,j) * dt
+    ! do i = i_litr_min, i_litr_max
```

Comment:
Hmmm.  This should just be removed since it's commented out. We don't really know if the new code is correct.

Reply:
OK @slevisconsulting didn't remove this comment out code, but let's just leave it with the TODO marker for it.

Reply:
if you look at "changes from all commits" you'll see that it's removed.